migrate/migrations/20210817132557-add-user-favorite.sql

```diff
@@ -0,0 +1,8 @@
+-- +migrate Up
+ALTER TABLE user_favorites
+    ADD COLUMN tgt_user_id UUID REFERENCES users(id) ON DELETE CASCADE,
+    ADD CONSTRAINT user_favorites_user_id_tgt_user_id_key UNIQUE(user_id, tgt_user_id);
+-- +migrate Down
+ALTER TABLE user_favorites
+    DROP CONSTRAINT user_favories_user_id_tgt_user_id_key, 
+    DROP COLUMN tgt_user_id;
```

Comment:
nit: end with newline
```suggestion
    DROP COLUMN tgt_user_id;

```

Reply:
addressed.